src/main/java/org/dependencytrack/resources/v1/AccessControlResource.java

```diff
@@ -65,7 +65,8 @@ public class AccessControlResource extends AlpineResource {
     @ApiOperation(
             value = "Returns the projects assigned to the specified team",
             response = String.class,
-            responseContainer = "List"
+            responseContainer = "List",
+            notes = "<p>Requires permission <b>ACCESS_MANAGEMENT</b></p>"
```

Comment:
I've read somewhere that **<strong>** should be preferred to **<b>** as it is semantic. It therefore carries more meaning and can be interpreted better on more device and cases (like readers for blind people).  
Also what happens if you remove the use of the paragraph tag ? [Examples described](https://github.com/swagger-api/swagger-core/wiki/Annotations#apioperation) on the swagger api do not show usage of paragraph tags.